src/Helper/ToUtfConverter.php

```diff
@@ -26,8 +26,9 @@ public function convert(string $html, string $contentType = ''): string
         } elseif (preg_match("!.*<meta.*charset=[\"']?[ \t]*([A-Za-z0-9-]{4,})[ \t]*[\"']!mi", $html, $metaContentType) === 1) {
             $encoding = $metaContentType[1];
         }
-        $encoding = strtolower($encoding);
+        $encoding = mb_strtolower($encoding);
```

Comment:
why you are changing this line?

Reply:
This is absolutely irrelevant in this case. In our project we are completely switching to multibyte versions of the string functions. Therefore I decided to use it here, because "ext-mbstring" extension is already connected anyway.